File: swexpertacademy/get-average-value-2071.cpp
#include <iostream>
#include <cmath>
#include "solutions.h"

using namespace std;

void get_average_value_2071() {

    int time;
    cin >> time;

    for(int i=0; i<time; i++) {
        double sum = 0;

        for(int j=0; j<10; j++) {
            int input;
            cin >> input;
            sum += input;
        }

        cout << "#" << i + 1 << " " << (int)round(sum/10) << endl;
    }
}

File: JavaProblemSolving/src/swexpertacademy/GetAverageValue2071.java
package swexpertacademy;

import java.util.Scanner;

public class GetAverageValue2071 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int t = scanner.nextInt();

        for(int i=1; i<=t; i++){
            int sum = 0;

            for(int j=0; j<10; j++){
                int input = scanner.nextInt();
                sum += input;
            }

            int answer = (int)Math.round((double)sum/10);

            System.out.println("#"+i+" "+answer);
        }
    }
}
